apps/docs/content/guides/realtime/realtime-user-presence.mdx

```diff
@@ -0,0 +1,19 @@
+---
+id: 'realtime-user-presence'
+title: 'Using Realtime Presence with Flutter'
+description: 'Track online users with Supabase Realtime Presence'
+sidebar_label: 'Videos'
+---
+
+Use Supabase Presence to display the currently online users on your Flutter application.
+
+Displaying the list of currently online users is a common feature for a real-time collaborative applications. Supabase Presence makes it easy to track users joining and leaving the session so that you can easily make a collaborative app.
```

Comment:
```suggestion
Displaying the list of currently online users is a common feature for real-time collaborative applications. Supabase Presence makes it easy to track users joining and leaving the session so that you can easily make a collaborative app.
```